components/UI/CookiesConsent.vue

```diff
@@ -14,7 +15,7 @@ export default defineNuxtComponent({
 
   computed: {
     vidoConfig(): VidoConfig {
-      return this.$vidoConfig(useRequestHeaders())
+      return useSiteStore().config || this.$vidoConfig(useRequestHeaders())
```

Comment:
It looks like we have some server code executing on client-side from config.ts:vidos() as we are trying to read from local JSON file.

Maybe we should review the way we get Configs. We could load it at a higher level (ex: layout) and then serve it through useNuxtData() caching system.

We could only use `config` from store once all the pages supports it.

Reply:
`this.$vidoConfig(useRequestHeaders()` should be removed.

Reply:
Not here as we are using this component globally in the App.
Getting `config` from store is only available for `/category/[id]` page for the moment, so it will be undefined on other pages, thus we have to rely on `$vidoConfig` for the moment.

Reply:
ok